src/openrct2/core/Guard.hpp

```diff
@@ -23,6 +23,7 @@ extern "C" {
 #endif // __cplusplus
 
 void openrct2_assert_va(bool expression, const char * message, va_list va);
+inline void openrct2_assert(bool expression, const char * message, ...);
```

Comment:
Without that line, you don't comply to C standard, I believe. GCC 4.9.4 says:
```
In file included from /home/olifre/gits/OpenRCT2/src/openrct2/drawing/drawing.c:19:0:
/home/olifre/gits/OpenRCT2/src/openrct2/drawing/../core/Guard.hpp:26:13: error: no previous declaration for ‘openrct2_assert’ [-Werror=missing-declarations]
 inline void openrct2_assert(bool expression, const char * message, ...)
             ^
cc1: all warnings being treated as errors
```
I guess the support for having only the definition and to completely omit the declaration is some gnu11 extension, but I don't think this is in any standard - or is it?